torchbenchmark/util/model.py

```diff
@@ -369,3 +370,104 @@ def inputs_convert(example_inputs):
             self.example_inputs = inputs_convert(self.example_inputs)
         else:
             warnings.warn(UserWarning(f"{model_name} example inputs doesn't convert to `channels_last`!"))       
+
+    def enable_fx_int8(self, quant_engine:str='x86'):
```

Comment:
`model.py` is pretty big already, and we would like to make the ownership clear. Can we place this function in a separate file, such as `fx_int8.py`?

Reply:
Yes, agree with you. I have move out the help functions of FX quantization into a new file `fx_int8.py`, but keep the `enable_fx_int8()` entry function in the `model.py`, because it should be a function of `Model`. How do you think?